sklearn/ensemble/_stacking.py

```diff
@@ -618,7 +644,16 @@ def predict(self, X, **predict_params):
             Predicted targets.
         """
         y_pred = super().predict(X, **predict_params)
-        return self._le.inverse_transform(y_pred)
+        if isinstance(self._label_encoder, list):
+            y_pred = np.array(
+                [
+                    self._label_encoder[target_idx].inverse_transform(target)
+                    for target_idx, target in enumerate(y_pred.T)
+                ]
+            ).T
```

Comment:
Is this right? 
```suggestion
            # Handle the multilabel-indicator and multiclass-multioutput cases
            y_pred = np.array(
                [
                    self._label_encoder[target_idx].inverse_transform(target)
                    for target_idx, target in enumerate(y_pred.T)
                ]
            ).T
```

Reply:
Yes that's correct!

Reply:
For the moment, it seems that we are only handling the multilabel-indicator case.
If we want to support the multiclass-multioutput, we need to add more tests and update the changelog.